os/overriding-systemd-units.md

```diff
@@ -0,0 +1,105 @@
+# Overriding systemd units
+
+There are two methods of overriding default CoreOS settings in unit files: copying the unit file from /usr/lib64/systemd/system to /etc/systemd/system and modifying the chosen settings. Alternatively, one can create a directory named unit.d/ within /etc/systemd/system and place a drop-in file name.conf there that only changes the specific settings one is interested in. Note that multiple such drop-in files are read if present.
+
+The advantage of the first method is that one easily overrides the complete unit, the default CoreOS unit is not parsed at all anymore. It has the disadvantage that improvements to the unit file supplied by CoreOS are not automatically incorporated on updates.
+
+The advantage of the second method is that one only overrides the settings one specifically wants, where updates to the original CoreOS unit automatically apply. This has the disadvantage that some future CoreOS updates might be incompatible with the local changes.
+
+Note that for drop-in files, if one wants to remove entries from a setting that is parsed as a list (and is not a dependency), such as ConditionPathExists= (or e.g. ExecStart= in service units), one needs to first clear the list before re-adding all entries except the one that is to be removed. See below for an example.
+
+This also applies for user instances of systemd, but with different locations for the unit files. See the section on unit load paths for further details.
+
+Let's review `fleet.service` unit /usr/lib64/systemd/system/fleet.service with the following contents:
```

Comment:
Maybe explain how you'd find this in the first place? `systemctl list-units | grep fleet`?